rasa/nlu/config.py

```diff
@@ -76,9 +77,21 @@ def component_config_from_pipeline(
     pipeline: List[Dict[Text, Any]],
     defaults: Optional[Dict[Text, Any]] = None,
 ) -> Dict[Text, Any]:
+    """Get the configuration of the `index`th component.
```

Comment:
Let's use [descriptive docstrings](https://www.notion.so/rasa/Python-Code-Conventions-26835ffd72484dd699f28fa693916d2e#14d49d4fe8ec4b3796eb27a721417174)
```suggestion
    """Gets the configuration of the `index`th component.
```